prism-vdr-specification.md

```diff
@@ -280,17 +306,47 @@ Design goals:
   <br>The Indexer is described for the general use case.
 
 
+## Considerations
+
+- **Public** -
+  Everything in the blockchain is public.
+  Although the content of the data can be encrypted or deferred to another place (e.g. IPFS)
+
+- **Permanent** -
+  All the information is considered permanent after the `Settlement time`.
+  At the time of writing `k=2160` (36 minutes).
+  But after submitting a transaction there is no operation to cancel.
+
+- **Size constraint** -
+  In Cardano, transactions was a max size.
+  So everything related to the transaction itself and the metadata in the PRISM Block containing the events MUST fit within the max size.
+  At the time of writing the max transaction size is `16,384 bytes`.
+
+- **Cryptographic & quantum-resistant** -
+  `CRYPTOGRAPHIC_CURVE` is the key type used for MASTER_KEY and VDR_KEY, which is not considered quantum-resistant.
+  However, the [`did:prism` DID method](https://github.com/input-output-hk/prism-did-method-spec/blob/main/w3c-spec/PRISM-method.md) like this protocol. It has the `ProtoProtocolVersionUpdate` event type that allows marking breaking changes in the specification by an official source.
+  This makes it possible to swap in post-quantum signature schemes in the future once standards mature.
+
+- **Notion of date-time** -
+  All events are associated to a Cardano transaction.
+  Those transaction can be associated with real date-time or a small period of time.
+  So this notion of real time can be used to prove that some information was available before some a date-time.
+
 ## Document History
 
 0. Initial draft.
-1. Restructuring the document, Fix typos and incorporate recommendations from reviews.
+1. Restructuring the document; Fix typos; Incorporate recommendations from reviews.
+2. Add sections considerations, custom encoding section; Add authors ???; Fix typos.
+10. ??? `VERSION 1.0`- Mark document as stabled.
 
 ## Authors and Contributors
 
 - **Fabio Pinheiro** <br>
   [Email: fabio.pinheiro@iohk.io](mailto:fabio.pinheiro@iohk.io) <br>
   [Github: FabioPinheiro](https://github.com/FabioPinheiro)
 
+- ???
```

Comment:
@yshyn-iohk @patextreme @mineme0110 @goncalo-frade-iohk you guys should add your names for all the contributions and discussions.